util/symexec.cpp

```diff
@@ -48,6 +48,9 @@ void sym_exec(State &s) {
 
     first = false;
   }
+  
+  if (s.foundReturn())
+    s.buildUB();
```

Comment:
Does with work with a basic block ending with unreachable as well?

Reply:
It does work, if there's no return then the return_domain takes the default value of false. Or if there is a return and there's an unreachable in between, the UB should be added for the assume, which should make the expression false when the input to the assume is 0.